ui/tutorial-renderer/src/main/res/drawable/img_tutorial_tips_learn.xml

```diff
@@ -0,0 +1,240 @@
+      android:fillColor="#E4E4E4"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M201.22,166.942L191.277,183.263C187.842,185.895 185.147,188.519 183.193,191.137C182.811,191.947 182.535,193.382 184.994,192.142C187.453,190.903 190.096,189.709 190.802,190.432C191.509,191.154 190.398,192.822 191.587,193.7C192.38,194.285 193.807,191.413 195.869,185.085L207.57,173.434L201.22,166.942Z"
+      android:fillColor="#F1CFB0"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M218.622,128.458L214.781,128.903C213.1,146.482 199.133,168.761 188.568,179.139L195.143,190.495C221.838,167.784 224.91,147.757 218.622,128.458Z"
+      android:fillColor="#C6201A"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M199.381,202.53C199.381,202.53 215.034,199.979 220.351,199.042C230.453,197.261 233.399,191.824 231.657,179.963L213.044,179.963C209.349,181.03 182.167,193.161 182.167,193.161C178.907,194.78 179.514,199.552 180.11,201.124C180.146,201.22 196.599,229.379 196.599,229.379L211.826,219.679L199.352,203.673C199.352,203.673 198.858,202.677 199.381,202.53Z"
+      android:fillType="evenOdd">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="204.268"
+          android:startX="187.969"
+          android:endY="226.78"
+          android:endX="210.481"
+          android:type="linear">
+        <item android:offset="0" android:color="#FFE6E6E6"/>
+        <item android:offset="1" android:color="#FFBEBEBE"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+  <path
+      android:pathData="M232.611,182.115L217.921,184.131L208.704,182.115L205.101,179.934L203.507,177.942C203.507,177.942 215.091,149.106 215.091,129.058L225.328,127.532C233.373,140.475 231.023,160.736 232.611,182.115Z"
+      android:fillColor="#DB2721"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M241.915,179.97C241.915,179.97 242.494,189.49 241.574,192.263C241.124,193.619 241.72,199.672 242.468,203.532C242.768,205.082 241.582,205.517 241.019,205.388C236.439,204.338 237.555,200.927 236.221,197.351C234.759,193.432 235.332,202.234 232.58,198.717C233.251,194.516 235.701,191.569 236.749,189.053C237.448,187.375 237.053,174.51 237.053,174.51L241.915,179.97Z"
+      android:fillColor="#F1CFB0"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M226.675,127.539C226.675,127.539 247.178,151.09 245.882,193.116L229.459,191.149C236.205,156.439 223.529,127.735 223.529,127.735L226.675,127.539Z"
+      android:fillType="evenOdd">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="142.962"
+          android:startX="232.724"
+          android:endY="184.509"
+          android:endX="247.604"
+          android:type="linear">
+        <item android:offset="0" android:color="#FFDB2721"/>
+        <item android:offset="1" android:color="#FFFF4133"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+</vector>
```

Comment:
this is way too complex for a vector image.

https://upday.github.io/blog/vector_drawables_optimisation/
https://developer.android.com/studio/write/vector-asset-studio.html

> The initial loading of a vector drawable can cost more CPU cycles than the corresponding raster image. Afterward, memory use and performance are similar between the two. We recommend that you limit a vector image to a maximum of 200 x 200 dp; otherwise, it can take too long to draw.